### ISSUE TITLE
Remove NFE usage in wavereform examples/docs (Trac #1557)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1549]
Migrated from https://code.icecube.wisc.edu/ticket/1557
```json
{
    "status": "closed", 
    "changetime": "2016-02-19T23:38:55", 
    "description": "", 
    "reporter": "jbraun", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1455925135769376", 
    "component": "combo reconstruction", 
    "summary": "Remove NFE usage in wavereform examples/docs", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-02-19T20:29:27", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160219T20:35:20: jbraun changed component from "cmake" to "icerec"

**Comment by icecube-trac** [comment]
Trac update at 20160219T23:38:55: 
* jbraun commented:

> In changeset 142166:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142166"
> Fix outdated example, update docs.  Closes #1549
> ```


* jbraun changed status from "new" to "closed"
* jbraun changed resolution from "" to "fixed"
